docs/docs/components.mdx

```diff
@@ -212,8 +212,14 @@ now implements its behavior.
       # Pre-Trained weights to be loaded
       model_weights: "rasa/LaBSE"
 
-      # An optional path to a specific directory to download and cache the pre-trained model weights.
-      # The `default` cache_dir is the same as https://huggingface.co/transformers/serialization.html#cache-directory .
+      # An optional path to a directory from which
+      # to load pre-trained model weights.
+      # If the requested model is not found in the
+      # directory, it will be downloaded and
+      # cached in this directory for future use.
+      # The default value of `cache_dir` is the
+      # constant `TRANSFORMERS_CACHE`, defined in
```

Comment:
Hmm kind of: I think this would take precedence over anything else if it were set, but if it's not set the default is different, see relevant code [here](https://github.com/huggingface/transformers/blob/v2.11.0/src/transformers/file_utils.py#L63-L76). I really don't know if this is splitting hairs, though, I'd be fine with saying environment variable if you are :)

Reply:
But you're definitely right that "constant" isn't the right term

Reply:
Thanks for linking to the code. The `TRANSFORMERS_CACHE` is indeed meant to be an environment variable that the use could configure. Let's go with "environment variable" instead of "constant"